spec/smoke/find/course_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "nokogiri"
+require "spec_helper_smoke"
+
+class HtmlParserIncluded < HTTParty::Parser
+  def html
+    Nokogiri::HTML(body)
+  end
+end
+
+class FindResultsPage
+  include HTTParty
+  parser HtmlParserIncluded
+
+  base_uri Settings.search_ui.base_url
+
+  Course = Struct.new(:url)
+
+  def self.call
+    new.courses
+  end
+
+  def response
+    @response ||= self.class.get("/results?age_group=primary&l=2&subject_codes%5B%5D=00").parsed_response
+  end
+
+  def courses
+    response.css(".app-search-results .app-search-results__item .app-search-result__item-title").map do |course|
+      Course.new(course.css(".govuk-link").first["href"])
+    end
```

Comment:
can you use the `data-qa="course__link"` instead

Reply:
done